apps/aecore/lib/aecore/structures/voting_tx.ex

```diff
@@ -0,0 +1,11 @@
+defmodule Aecore.Structures.VotingTx do
+  alias Aecore.Structures.VotingTx
+
+  @type t :: %VotingTx{
```

Comment:
`fee`, `nonce` and `from_acc` can be moved in here, as this duplicated code and is present in all sub types